global:
  foc_bus_voltage: 100
  serial_router_interface: eth3
chains:
  -
    eth1:
      -
        name: motorR0
        foc_max_effort: 100
        foc_integrator_limit: 20
        foc_stator_offset: 1580
        pole_pairs: 8
        foc_kp: -20
        foc_ki: -400
        enable_mosfets: true
        damping: -1
        resistance: 9.3
      -
        name: motorR1
        foc_max_effort: 20
        foc_integrator_limit: 20
        foc_stator_offset: 915
        pole_pairs: 12
        foc_kp: -3
        foc_ki: -100
        enable_mosfets: true
        damping: -1
        resistance: 0.46
      -
        name: motorR2
        foc_max_effort: 20
        foc_integrator_limit: 20
        foc_stator_offset: 952
        pole_pairs: 8
        foc_kp: -20
        foc_ki: -400
        enable_mosfets: true
        damping: 1
        resistance: 5.52
      -
        name: motorR3
        foc_max_effort: 20
        foc_integrator_limit: 20
        foc_stator_offset: 921
        pole_pairs: 12
        foc_kp: -3
        foc_ki: -100
        enable_mosfets: true
        damping: -1
        resistance: 0.46
      -
        name: motorR4
        foc_max_effort: 20
        foc_integrator_limit: 20
        foc_stator_offset: 1397
        pole_pairs: 8
        foc_kp: -20
        foc_ki: -400
        enable_mosfets: true
        damping: 1
        resistance: 3.85
      -
        name: motorR5
        foc_max_effort: 20
        foc_integrator_limit: 20
        foc_stator_offset: 108
        pole_pairs: 8
        foc_kp: -20
        foc_ki: -400    
        enable_mosfets: true
        damping: 1
        resistance: 3.85
  -
    eth2:
      -
        name: motorL0
        foc_max_effort: 100
        foc_integrator_limit: 20
        foc_stator_offset: 554
        pole_pairs: 8
        foc_kp: -20
        foc_ki: -400
        enable_mosfets: true
        damping: -1
        resistance: 9.3
      -
        name: motorL1
        foc_max_effort: 20
        foc_integrator_limit: 20
        foc_stator_offset: 49
        pole_pairs: 12
        foc_kp: -3
        foc_ki: -100
        enable_mosfets: true
        damping: 1
        resistance: 0.46
      -
        name: motorL2
        foc_max_effort: 20
        foc_integrator_limit: 20
        foc_stator_offset: 1293
        pole_pairs: 8
        foc_kp: -20
        foc_ki: -400
        enable_mosfets: true
        damping: 1
        resistance: 5.52
      -
        name: motorL3
        foc_max_effort: 20
        foc_integrator_limit: 20
        foc_stator_offset: 1164
        pole_pairs: 12
        foc_kp: -3
        foc_ki: -100
        enable_mosfets: true
        damping: 1
        resistance: 0.46
      -
        name: motorL4
        foc_max_effort: 20
        foc_integrator_limit: 20
        foc_stator_offset: 118
        pole_pairs: 8
        foc_kp: -20
        foc_ki: -400
        enable_mosfets: true
        damping: -1
        resistance: 3.85
      -
        name: motorL5
        foc_max_effort: 20
        foc_integrator_limit: 20
        foc_stator_offset: 1525
        pole_pairs: 8
        foc_kp: -20
        foc_ki: -400     
        enable_mosfets: true
        damping: -1
        resistance: 3.85
